<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogFilterDialog</class>
 <widget class="QDialog" name="LogFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log Filter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Log Levels</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="informationsCheckBox">
        <property name="text">
         <string>Informations</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="warningsCheckBox">
        <property name="text">
         <string>Warnings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="errorsCheckBox">
        <property name="text">
         <string>Errors</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Tags</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="allTagsRadioButton">
        <property name="text">
         <string>All Tags</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="allRegisteredTagsRadioButton">
        <property name="text">
         <string>All Registered Tags</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="allUnregisteredTagsRadioButton">
        <property name="text">
         <string>All Unregistered Tags</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QRadioButton" name="registeredTagRadioButton">
          <property name="text">
           <string>Registered Tag</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="registeredTagComboBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="anyTagRadioButton">
          <property name="text">
           <string>Any Tag</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="anyTagLineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPushButton">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>LogFilterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
